app/views/News.js

```diff
@@ -168,10 +171,14 @@ const styles = StyleSheet.create({
   },
   singleNewsHead: {
     height: 48,
+    backgroundColor: 'rgba(255,255,255,0.6)',
+    borderTopLeftRadius: 12,
```

Comment:
Since you are changing visuals, can you provide screenshots?

Reply:
![Simulator Screen Shot - iPhone 8 - 2020-04-30 at 14 00 07](https://user-images.githubusercontent.com/62654805/80689773-700bf180-8aeb-11ea-8a6f-5a4f504c53f2.png)